src/cljdoc/render/layout.clj

```diff
@@ -10,11 +10,15 @@
 
 (defn highlight-js-customization []
   [:script
-   (->> ["hljs.configure({languages: ['clj', 'cljs']});"
-         "hljs.initHighlightingOnLoad();"
-         "hljs.registerLanguage('cljs', function (hljs) { return hljs.getLanguage('clj') });"]
-        (string/join "\n")
-        (hiccup/raw))])
+   (hiccup/raw
+     "hljs.registerLanguage('cljs', function (hljs) { return hljs.getLanguage('clj') });
+      addEventListener('load', function() {
+        var code = document.querySelectorAll('pre code');
+        code.forEach(function (el) {
+          el.classList.add('language-clj');
+          hljs.highlightBlock(el);
+        });
+      });")])
```

Comment:
This doesn't sit quite right with since it will render any code block with Clojure highlighting. Could you give some examples for what this is necessary? 

If it is about code blocks in docstrings where triple-backtick code blocks are inconvenient then maybe we could implement a special markdown rendering function that adds the class to code elements. This would make the change not affect code blocks in articles where it's probably more common to have some non-Clojure code snippets.